tests/Unit/Models/PostModelTest.php

```diff
@@ -60,7 +60,9 @@ public function testSiblingRelationship()
      */
     public function testBlinkPayload()
     {
-        $post = factory(Post::class)->create();
+        $post = factory(Post::class)->create([
+            'school_id' => 'Example School ID',
```

Comment:
I'm confused about how this `school_id` gets linked up to `San Dimas High School`?

Reply:
It's because we're always returning that school name via [GraphQL mock](https://github.com/DoSomething/rogue/blob/d77c9b879e168deaf5118962c2da4fabcb2c4d75/tests/WithMocks.php#L60) (in honor of Bill and Ted 3 release this summer).

Reply:
Ah gotcha! What a great lil' easter egg 😂